README.md

```diff
@@ -0,0 +1,219 @@
+# BoutrosLab.plotting.general
+
+1. [Description](https://github.com/uclahs-cds/public-R-BoutrosLab-plotting-general#Description)
+2. [Included plots](https://github.com/uclahs-cds/public-R-BoutrosLab-plotting-general#Included-plots)
+3. [Installation](https://github.com/uclahs-cds/public-R-BoutrosLab-plotting-general#Installation)
+4. [Function reference](https://github.com/uclahs-cds/public-R-BoutrosLab-plotting-general#Function-reference)
+5. [Resources](https://github.com/uclahs-cds/public-R-BoutrosLab-plotting-general#Resources)
+6. [Getting help](https://github.com/uclahs-cds/public-R-BoutrosLab-plotting-general#Getting-help)
+7. [Contributors](https://github.com/uclahs-cds/public-R-BoutrosLab-plotting-general#Contributors)
+8. [Citation information](https://github.com/uclahs-cds/public-R-BoutrosLab-plotting-general#Citation-information)
```

Comment:
to link to the headings on the same page, you use the one `#` and then using all lowercase, with hyphens `-` as spaces. I don't recall for numbers or special characters what you do, but you can just use the link to the side of the heading like you seemed to have used and just delete the URL before the heading. 
```suggestion
1. [Description](#description)
2. [Included plots](#included-plots)
3. [Installation](#installation)
4. [Function reference](#function-reference)
5. [Resources](#resources)
6. [Getting help](#getting-help)
7. [Contributors](#contributors)
8. [Citation information](#citation-information)
```